### PR TITLE
feat: tag trigger example.

### PR DESCRIPTION
**Pull request by wdstar**
## Context

We add a tag triggered job example.

## References

- https://github.com/screwdriver-cd/screwdriver/issues/823

### CODE REVIEW

**Review by yuichi10 — APPROVED** [review]
LGTM

**Review by s-yoshika — APPROVED** [review]

File: screwdriver.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+jobs:
+    test-job:
```
Comment (tk3fftk) [inline screwdriver.yaml:2]: Do we need this `test-job`?

Comment (wdstar) [reply screwdriver.yaml:2]: Not necessarily for this example, but I want to leave a best practice job.

**Review by tk3fftk — APPROVED** [review]
LGTM if CI is set and passed

**Pull request merged**